script.js

```diff
@@ -0,0 +1,88 @@
+// declare an empty array to store book list
+let bookList = [];
+
+// generating bookobject
+
+function bookObject(bookTitle, authorName) {
+  const books = {
+    title: bookTitle,
+    author: authorName,
+  };
+
+  bookList.push(books);
```

Comment:
- [OPTIONAL] You have done very good work getting this far.  As an improvement, you should prefer methods that don't mutate the array state like `concat()` over `push()`. You can checkout out this [resource](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) to read abut array methods. 👍🏾